myning/tui/chapter/__init__.py

```diff
@@ -195,13 +195,13 @@ def get_labels_and_hotkeys(options: list[OptionLabel]):
             if hotkey and hotkey_index is not None:
                 hotkeys[hotkey] = option_index
                 text_option.stylize("underline", hotkey_index, hotkey_index + 1)
-                option_arr[text_option_index] = text_option
+                option_arr.label[text_option_index] = text_option
 
-        labels.append(option_arr)
+        labels.append(option_arr.label)
     return labels, hotkeys
 
 
-def get_hotkey(label: str, hotkeys: dict[str, int]):
+def get_hotkey(label: str, hotkeys: dict[str, int]) -> tuple[str | None, int | None]:
```

Comment:
if we want to typehint, the correct return type would be `tuple[str, int] | tuple[None, None]` but how about we just leave it implicit?

Reply:
I fixed the typing